Jenkinsfile

```diff
@@ -0,0 +1,38 @@
+@Library('jenkins-joylib@v1.0.0') _
+
+pipeline {
+
+    agent {
+        label joyCommonLabels(image_ver: '18.4.0')
```

Comment:
Is there any way we can use the BASE_IMAGE_UUID stuff in Makefile to auto-fill the value of this? Does it mean that every time we bump an image, we have to now change two files per repository?

Reply:
> Is there any way we can use the BASE_IMAGE_UUID stuff in Makefile to auto-fill the value of this?

I'll copy my answer from chat: The answer would be "write some groovy code",but figuring it out from BASE_IMAGE in the Makefile is possible in principle.  That only helps for the minority of repos that build images.

> Does it mean that every time we bump an image, we have to now change two files per repository?

Yes! but, the previous state of affairs was:
 * Change a file in a repo; push to a branch
 * clone the jenkins freestyle job; muck with the labels and point it at your branch so you could test
 * Send a pr/cr/whatever 
 * Make sure you change the original jenkins job at about the same time you land your PR
 * deleting your testing clone job
 * Make sure you don't do any of this near a release or everything will break

So I think two files changed in one atomic commit is a net improvement